tests/snapshots/src/main/generated/BaseByteRenderer.scala

```diff
@@ -23,39 +23,57 @@ import upickle.core.{ArrVisitor, ObjVisitor}
   */
 class BaseByteRenderer[T <: upickle.core.ByteOps.Output]
 //    ^^^^^^^^^^^^^^^^ definition semanticdb maven maven/com.lihaoyi/ujson_2.13 1.4.0 ujson/BaseByteRenderer#
-//                     documentation ```scala\nclass BaseByteRenderer[T <: Output]\n```
+//                     display_name BaseByteRenderer
+//                     signature_documentation scala class BaseByteRenderer[T <: Output]
+//                     kind Class
 //                     relationship is_reference semanticdb maven maven/com.lihaoyi/ujson_2.13 1.4.0 ujson/BaseByteRenderer.
 //    ________________ synthetic_definition semanticdb maven maven/com.lihaoyi/ujson_2.13 1.4.0 ujson/BaseByteRenderer.
-//                     documentation ```scala\nobject BaseByteRenderer\n```
+//                     display_name BaseByteRenderer
+//                     signature_documentation scala object BaseByteRenderer
+//                     kind Object
 //                     relationship is_definition semanticdb maven maven/com.lihaoyi/ujson_2.13 1.4.0 ujson/BaseByteRenderer#
 //                     ^ definition semanticdb maven maven/com.lihaoyi/ujson_2.13 1.4.0 ujson/BaseByteRenderer#[T]
-//                       documentation ```scala\nT <: Output\n```
+//                       display_name T
+//                       signature_documentation scala T <: Output
+//                       kind TypeParameter
 //                          ^^^^^^^ reference semanticdb maven . . upickle/
 //                                  ^^^^ reference semanticdb maven . . upickle/core/
 //                                       ^^^^^^^ reference semanticdb maven maven/com.lihaoyi/upickle-core_2.13 1.4.0 upickle/core/ByteOps.
 //                                               ^^^^^^ reference semanticdb maven maven/com.lihaoyi/upickle-core_2.13 1.4.0 upickle/core/ByteOps.Output#
                       (out: T,
 //                     ^^^ definition semanticdb maven maven/com.lihaoyi/ujson_2.13 1.4.0 ujson/BaseByteRenderer#out.
-//                         documentation ```scala\nprivate[this] val out: T\n```
+//                         display_name out
+//                         signature_documentation scala private[this] val out: T
+//                         kind Method
```

Comment:
This for instance should be a Field rather than a Method, but I believe this is due to a bug in semanticdb-scalac rather than scip-semanticdb.

Reply:
I'm just going to confer with the council of Semanticdb experts to see if this is intentional due to Scalac class encoding, or accidental.